server/src/test/java/keywhiz/commands/DropDeletedSecretsCommandTest.java

```diff
@@ -40,7 +40,7 @@
 
   // an undeleted secret
   private SecretSeries series1 =
-      SecretSeries.of(1, "secret1", "desc1", january18, "creator", january19, "updater", null, null,
+      SecretSeries.of(1, "secret1", null, "desc1", january18, "creator", january19, "updater", null, null,
```

Comment:
Is the owner field set to `null` in these tests until the database schema is updated (as described in a comment above)?

Reply:
It depends on the test. For most of these the values are irrelevant and were just copied from other tests. Without builder support (which I was debating on adding) it makes it difficult to create mostly-empty POJOs for testing. For integration tests where there's actual calls to a local database then yeah, it totally makes sense to provide a value (assuming it's relevant to the test), but that'll come when we have our updated schema.